pkg/antctl/raw/multicluster/init.go

```diff
@@ -0,0 +1,98 @@
+// Copyright 2022 Antrea Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package multicluster
+
+import (
+	"fmt"
+	"strings"
+
+	"antrea.io/antrea/pkg/antctl/raw/multicluster/common"
+	"github.com/spf13/cobra"
+)
+
+type initOptions struct {
+	namespace  string
+	clusterSet string
+	clusterID  string
+}
+
+var initOpts *initOptions
+
+func (o *initOptions) validate() error {
+	if o.namespace == "" {
+		return fmt.Errorf("Namespace is required")
+	}
+	if o.clusterSet == "" {
+		return fmt.Errorf("ClusterSet is required")
+	}
+	if o.clusterID == "" {
+		return fmt.Errorf("ClusterID is required")
+	}
+	return nil
+}
+
+var initExample = strings.Trim(`
+# Initialize ClusterSet in the given Namespace of the leader cluster.
+  $ antctl mc init --namespace antrea-multicluster --clusterset test-clusterset --clusterid test-cluster-north
```

Comment:
I feel better to remove "test-" from cluster IDs, so: cluster-north, cluster-east.

Reply:
Removed all "test-" from examples.